### PR TITLE
added more tools

### PR DESCRIPTION
**Pull request by RRozz**
tools added include: 
new exp modifier -- write expressions to change exp of monsters based on their levels
&
item shop validator -- remove items from a merchant's shop records if the items aren't in the open ItemInfo.shn, and optionally sort the items in a merchant's shop so that all non-empty item slots are moved to the beginning of the shop

### CODE REVIEW

**Comment by skeleten** [comment]
Heya,

first of all, thanks for your contribution. The failing CI tests seems to be an invalid configuration and not on your side.

It seems however that you made multiple pull requests with the same name. Could you consolidate these into a single one?

Thanks,
- skeleten

**Comment by RRozz** [comment]
Hey,
This was my first time contributing to GitHub and I'm pretty
confused/overwhelmed;
How do I combine my pull requests, by making a new branch?
(the first pull request was an accident, the second one had the actual
additions in it)

**Comment by skeleten** [comment]
If all the relevant changes are in this PR we should be fine by just merging it and closing the other :)

**Comment by RRozz** [comment]
looks right, sweet :)

File: src/SHNDecryptor/More Tools/MoreTools_help.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            mtAbout.ShowDialog();
+        }
+
+        private void MoreTools_help_Load(object sender, EventArgs e)
+        {
+            String tmp = "";
+            switch (mySubject)
+            {
+                case 1:
+                    {
```
Comment (skeleten) [inline src/SHNDecryptor/More Tools/MoreTools_help.cs:37]: You can create multiline strings like this:

```csharp

string foo = @"Line 1
Line 2
Line3";
```

**Comment by RRozz** [comment]
oh cool, thanks
i'll take any other advice / constructive criticism you have~ i could use it

**Pull request merged**